_co-op-companies/N-rbc.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Royal Bank of Canada"
+identification: "rbc"
+description: "The Royal Bank of Canada is a bank that provides commerce services."
+link: "https://jobs.rbc.com/ca/en"
+image: "assets/img/logos/rbc.png"
+width: "100px"
+complete: true
+members:
+    - name: "Shenyun Wang"
+      term: "Work Term: Fall 2018, Summer 2018 "
+      tools: "Angular6, SpringBoot"
+      image: "/assets/img/co-op/shenyun.jpg"
+---
```

Comment:
looks like you included files from your rbc branch in the master of code branch. when creating a new branch you should always branch off of master `git checkout master` then pulling masters version with `git pull` and then creating your new branch. For this pull request you can just remove the rbc related files and recover the config file to avoid merge conflicts once your rbc pull request is merged :)

Reply:
Sorry about that, I'll make sure to do that next time, thank you! For now I'll delete the rbc-related files.